tests/Doctrine/Tests/DBAL/Functional/WriteTest.php

```diff
@@ -285,4 +285,44 @@ public function testEmptyIdentityInsert()
 
     }
 
+    /**
+     * @group DBAL-2688
+     */
+    public function testUpdateWhereIsNull()
+    {
+        $this->_conn->insert(
+            'write_table',
+            ['test_int' => '30', 'test_string' => null],
+            ['test_string' => 'string', 'test_int' => 'integer']
+        );
+
+        $data = $this->_conn->fetchColumn('SELECT COUNT(*) FROM write_table WHERE test_int = 30');
+
+        $this->assertEquals(1, $data);
```

Comment:
Which type of assert choose isn't related to CS, it depends on the type of data you need to check.